tools/shared.py

```diff
@@ -895,10 +895,10 @@ def get_vanilla_file():
       logging.debug('failed to use vanilla file, will re-check: ' + str(e))
       is_vanilla = check_vanilla()
     temp_cache = None
+    os.environ['EMCC_WASM_BACKEND'] = str(is_vanilla)
```

Comment:
Sorry, I don't understand, and I also don't understand where "below" is (below in the file? below in the review comment responses?).